skimage/transform/radon_transform.py

```diff
@@ -23,10 +9,10 @@
 from warnings import warn
 
 
-__all__ = ["radon", "iradon", "iradon_sart"]
+__all__ = ['radon', 'order_angles_golden_ratio', 'iradon', 'iradon_sart']
```

Comment:
I'm not sure... Indeed, it's not used internally. To me, the user has the choice to use it or not in his/her workflow.